lib/spatial_features/importers/base.rb

```diff
@@ -19,6 +19,12 @@ def cache_key
         @cache_key ||= Digest::MD5.hexdigest(@data)
       end
 
+      # factory method that should always be used when creating importers
+      # returns an array of Importer::* objects
```

Comment:
This second line of the comment seems to only apply to the `File` importer. What if we called this `create_all` or `build_all` and then it would always return an array.

Reply:
Good call about the return types.  `Base::create` should have returned an array all along to show the API we expect.